docs/source/TaskEngineWorkflows.md

```diff
@@ -538,34 +589,41 @@ The schedule_interval can be set via an api call. (where x is time in seconds)
 ```
 
 A jobs `run_at` attribute can be set in multiple ways and defaults to the time it was created at.
+Format: yyyy-mm-ddThh:mm:ss
 
 1. When submitting a job
 
-`post /job/:job_id`  
+`post '/job/:job_id'`
 
 ```json
 {
   "client": "demo-client",
   "job": {
     "workflow": "vodcapture",
-    "run_at": "1970-01-01T00:00:00"
+    "run_at": "2019-06-06T10:00:00.000"
   }
+}
 ```
 
+If the job's `run_at` time is in the future, a log will be added to indicate such. The log will be viewable from the job's log page.
+This is particularly useful when the run_at time is changed upon submission, due to a clip end time being in the past.
+
+ex: `Job will run at: "2019-06-06T10:00:00.000"`
+
 2. When updating an existing job
 
-`put /jobs/:job_id`
+`put '/jobs/:job_id'`
 
 ```json
 {
   "client": "demo-client",
-  "run_at": "1970-01-01T00:00:00"
+  "run_at": "2019-06-06T10:00:00.000"
 }
 ```
 
 3. When submitting a capture with a clip end time in the future
 
```

Comment:
` the future. Unless t` 
use a comma there not a fullstop

Reply:
Or put it in brackets.

Reply:
Reworded the sentence to use:
the end time of the clip which is furthest in the future. The exception to this is if the `run_at` time (if specified) is further in the future than the end time.